src/pretalx/settings.py

```diff
@@ -452,6 +471,12 @@ def reduce_dict(data):
         }
     },
 }
+email_level = config.get('logging', 'email_level', fallback='ERROR') or 'ERROR'
```

Comment:
in which case do you expect the `or 'ERROR'` to be executed? someone sets a value (and thus circumvents `fallback=`), buuut the value is falsy?